src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
-Title: Managed solutions under SLA
+Title: Managed solutions
 
-# Managed solutions under SLA
+# Managed solutions
 
-[Juju experts](https://jujucharms.com/experts) may designate a Juju
-model as a managed, supported solution by setting an SLA.
+A Juju Expert is a partner backed by professional support at Canonical Ltd.
+Such a partner deploys and supports an infrastructure based on Juju. We call
+this a *managed solution*.
 
-There are three support levels that may be set on a model:
+Any support requests beyond the build, workload, and operation of a particular
+solution can be escalated to Canonical by means of their Ubuntu Advantage
+support framework. Prime examples of such requests are for issues relating to
+Juju itself, Ubuntu-specific software, or the operating system kernel. The
+escalated support case will be treated by Canonical at the same support level
+currently ascribed to the partner. Hence, support level names used by both a
+Juju Expert and by Canonical are the same:
 
-- Essential
-- Standard
-- Advanced
+ - Essential
+ - Standard
+ - Advanced
 
-Our Juju expert partners support the managed solution itself, including its
-workloads and operation. Support for Juju and Ubuntu is provided by
-Canonical. See [Ubuntu Advantage for Servers](https://www.ubuntu.com/support/plans-and-pricing#server)
-for more information on Canonical support levels.
+See [Ubuntu Advantage for Servers][ubuntu-advantage-servers] for more
+information on Canonical support levels.
 
-Juju experts will provide details on the solution-specific support provided
-under each SLA, with budget estimates.
+The Juju Expert will provide details such as support levels and costs on a
+per-solution basis but ultimately an SLA will be conferred upon a specific Juju
+model (see [Models][models]).
 
-## Setting an SLA on a model
+Consult the external [Juju Experts][juju-experts] page to find a managed
+solution partner for your project.
 
-An SLA is set on a model with a monthly budget limit. Support costs will not be
-billed for more than this amount. If metered costs exceed this amount, the
-budget limit will need to be increased in order to continue to access support.
+## Requirements
 
-Set an Essential SLA on the current model _marketdata_, with a monthly budget
-limit of $100 USD:
+The following requirements need to be met before setting an SLA for the first
+time:
+
+ 1. The Juju operator will need an [Ubuntu SSO][ubuntu-sso] account.
```

Comment:
but is the operator the expert or the end user?

Reply:
The term "Juju operator" is what the documentation uses to refer to the person who operates Juju. It is distinguished from a "Juju user" (`add-user`). I can just say "you" if that is clearer. We do not use the term "end user" in the documentation as it is too vague.

Reply:
Sorry, maybe my question is more is that if the Expert is the Juju operator I'd not put it here. Again, it's more details for end users they don't need to know or worry about. Just that they have a way to get expertise and Canonical has their back. They're trustworthy and we're attaching our name/trust to them.

Reply:
The Juju operator is the customer. Everything in this document is for the customer, except the bit about escalation being done by the Expert.